### DISCUSSION TITLE
eXo openId configuration getting issue

### DISCUSSION THREAD

**Discussion by Rajamuthukrishna** [exoplatform/exo-community#53] [Q&A]
Hello Team,
I try to configure the eXo openId configuration with keyclock, once click the **sign in with single account** option getting some issue. shared the logs and configuration files, kindly help to resolve the issue.

<img width="1018" alt="keyclock_01" src="https://github.com/exoplatform/exo-community/assets/112479162/e49319f8-127b-49ee-9b3f-7f0b7add6703">
<img width="960" alt="keyclock_02" src="https://github.com/exoplatform/exo-community/assets/112479162/86e84cb0-aeed-4f4f-aaed-68c020d126aa">
<img width="960" alt="keyclock_03" src="https://github.com/exoplatform/exo-community/assets/112479162/ae62b0ac-7f4c-4d0d-ba06-248284e1a0c8">
<img width="960" alt="keyclock_04" src="https://github.com/exoplatform/exo-community/assets/112479162/773c335d-dcb5-4eb6-9f9b-f823e29615de">
<img width="960" alt="keyclock_05" src="https://github.com/exoplatform/exo-community/assets/112479162/e454611d-9b91-4d4f-90df-76273b5a03d5">
<img width="960" alt="exo-pic1" src="https://github.com/exoplatform/exo-community/assets/112479162/41ed3daf-b156-45f8-a704-9b1261d07214">
<img width="960" alt="exo-pic2" src="https://github.com/exoplatform/exo-community/assets/112479162/e78bcf7e-06fc-4b5e-93cf-ee141d818ddb">
<img width="960" alt="exo-pic3" src="https://github.com/exoplatform/exo-community/assets/112479162/4473fb6f-91ca-4c1f-86e1-012bc1066e24">

docker-compose:
[docker-compose.txt](https://github.com/exoplatform/exo-community/files/13435066/docker-compose.txt)

nginx.conf:
[nginx.txt](https://github.com/exoplatform/exo-community/files/13435089/nginx.txt)

exo.properties
[exo.txt](https://github.com/exoplatform/exo-community/files/13435095/exo.txt)

eXo logs:
[eXo-logs.txt](https://github.com/exoplatform/exo-community/files/13435103/eXo-logs.txt)

Redirecting URL having issue:
**http://localhost/portal/null**?response_type=code&client_id=exo-openid&scope=openid%20profile%20email&redirect_uri=http://localhost/portal/openidAuth&state=1951980419119501194

Regards,
Raja

**Comment by rdenarie** [comment]
Hello @Rajamuthukrishna 
I just answer on #44, so I close this one as duplicated
